index.html

```diff
@@ -11525,7 +11525,8 @@ <h2>Acknowledgements</h2>
 <!-- Simon Stewart --> <a href=http://www.rocketpoweredjetpants.com/>Simon Stewart</a>,
 <!-- Sri Harsha --> Sri Harsha,
 <!-- Titus Fortner --> Titus Fortner,
-<!-- Vangelis Katsikaros --> and Vangelis Katsikaros.
+<!-- Vangelis Katsikaros --> Vangelis Katsikaros,
+<!-- Zhizhen Ye --> and Zhizhen Ye.
```

Comment:
Please revert these changes. Originally new entries were added for substantial larger contributions but we may consider to remove them. Thanks.

Reply:
I would then defer to @AutomatedTester and @shs96c to decide, given that they are the actual editors of the specification.